packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.less

```diff
@@ -0,0 +1,52 @@
+@import (reference) "node_modules/@nova-ui/bits/sdk/less/nui-framework-variables";
+@import (reference) "@nova-ui/bits/sdk/less/nui-framework-variables.less";
+@import (reference) "@nova-ui/bits/sdk/less/mixins.less";
+
+@nui-legend-deemphasized-opacity: 0.3;
+@nui-legend-series-content-margin: 3px;
+
+.legend-description-container {
+    & .legend-description {
+        // line-height: @nui-line-height-default;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        padding-left: @nui-space-xs;
+        padding-right: @nui-space-xs;
+        min-width: 130px;
+
+        // &-container {
+        //     overflow: hidden;
+        // }
+
+        &-primary {
+            font-weight: @nui-font-weight-semibold;
+            .setCssVariable(color, nui-color-text-default);
+            &.link {
+                .setCssVariable(color, nui-color-text-link);
+                &:hover {
+                    text-decoration: underline;
+                }
+            }
+        }
+
+        &-secondary {
+            .setCssVariable(color, nui-color-text-secondary);
+            margin-top: -1px;
+            &.link {
+                .setCssVariable(color, nui-color-text-link);
+                &:hover {
+                    text-decoration: underline;
+                }
```

Comment:
I think we might be able to use the typography provided in bits for this: https://github.com/solarwinds/nova/blob/main/packages/bits/src/styles/nui-framework-typography.less#L223-L230

Reply:
@import (reference) "@nova-ui/bits/sdk/less/nui-framework-typography.less";